examples/packager.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python3
+#
+# This work is licensed under the GNU GPLv2 or later.
+# See the COPYING file in the top-level directory.
+
+# packager.py
+# CLI BZ routines for stuff that I do
+
+from argparse import ArgumentParser
+import bugzilla
+import sys
+
+
+def main():
+
+    parser = ArgumentParser(prog="rhbz")
+
+    # flags to set behavior
+    parser.add_argument(
+        "-e",
+        "--epel",
+        action="store_true",
+        dest="epel",
+        help="bool: file against EPEL product",
+        default=False,
+    )
+
+    # flags to set behavior
+    parser.add_argument(
+        "-n",
+        "--newpackage",
+        action="store_true",
+        dest="newpackage",
+        help="bool: submit a new package for review",
+        default=False,
+    )
+
+    # variables
+    parser.add_argument(
+        "-c",
+        "--component",
+        required=True,
+        dest="component",
+        help="file against this component (usually a package)",
+        type=str,
+    )
+
+    parser.add_argument(
+        "-p",
+        "--packager",
+        dest="packager",
+        help="FAS if reporter is Fedora packager",
+    )
+
+    args = parser.parse_args()
+
+    # some sanity checks
+    if args.epel == args.newpackage:
```

Comment:
I would assume you meant to use `and` instead of `==`, because the latter will evaluate to `True`, if both values are `False`.

```suggestion
    if args.epel and args.newpackage:
```